pkg/services/live/live.go

```diff
@@ -105,9 +105,10 @@ func ProvideService(plugCtxProvider *plugincontext.Provider, cfg *setting.Cfg, r
 	// things. For example Node allows to publish messages to channels from server
 	// side with its Publish method.
 	node, err := centrifuge.New(centrifuge.Config{
-		LogHandler:       handleLog,
-		LogLevel:         centrifuge.LogLevelError,
-		MetricsNamespace: "grafana_live",
+		LogHandler:         handleLog,
+		LogLevel:           centrifuge.LogLevelError,
+		MetricsNamespace:   "grafana_live",
+		ClientQueueMaxSize: 4194304, // 4MB
```

Comment:
i don't know anything about centrifuge but i wonder if we can improve the error msg somehow? the message was something like:

```
DEBUG[09-05|16:53:34] Client disconnected                      logger=live user=1 client=04ec8978-46b4-4400-bf39-905868a8c22c reason=slow elapsed=1.299456625s
```

which seems hard to connect to this queue size config. not a big deal, but may help someone in the future if we happen to cross the 4MB limit